services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteSpec.groovy

```diff
@@ -420,6 +420,63 @@ class AutoRerouteSpec extends HealthCheckSpecification {
         flowHelper.deleteFlow(flow.id)
     }
 
+    def "System doesn't reroute flow to a path with not enough bandwidth available"() {
+        given: "A flow with diverse path available"
+        def switchPair = topologyHelper.getAllNeighboringSwitchPairs().find {
+            it.paths.unique(false) { a, b -> a.intersect(b) == [] ? 1 : 0 }.size() >= 2
+        } ?: assumeTrue("No suiting switches found", false)
+
+        def flow = flowHelper.randomFlow(switchPair)
+        flowHelper.addFlow(flow)
+
+        def currentPath = pathHelper.convert(northbound.getFlowPath(flow.id))
+        def altPath = switchPair.paths.find { it != currentPath }
+        List<PathNode> broughtDownPorts = []
+        switchPair.paths.findAll { it != currentPath && it != altPath && it.first().portNo != altPath.first().portNo }
```

Comment:
What is happening here? This does not fall under 'A flow with diverse path available' description and there are no other comments. I'm lost at this point